src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+| process.runtime.dotnet.**jit.compilation_time**       | The amount of time the JIT compiler has spent compiling methods since the process start.                                                                                                                                                                               | `ns`        | ObservableCounter | `Int64`    |                  |                  |
+
+The APIs used to retrieve the values are:
+
+* [JitInfo.GetCompiledILBytes](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledilbytes?view=net-6.0#system-runtime-jitinfo-getcompiledilbytes(system-boolean)):
+  Gets the number of bytes of intermediate language that have been compiled.
+The scope of this value is global. The same applies for other JIT related metrics.
+
+* [JitInfo.GetCompiledMethodCount](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledmethodcount?view=net-6.0#system-runtime-jitinfo-getcompiledmethodcount(system-boolean)):
+  Gets the number of methods that have been compiled.
+
+* [JitInfo.GetCompilationTime](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompilationtime?view=net-6.0#system-runtime-jitinfo-getcompilationtime(system-boolean)):
+  Gets the amount of time the JIT Compiler has spent compiling methods.
+
+### Threading related metrics
+
+These metrics are only available when targeting .NET Core 3.1 or later.
+
+| Name                                                         | Description                                                                                                                                                                                                                                                                         | Units       | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|--------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**monitor.lock_contention.count**     | The number of times there was contention when trying to acquire a monitor lock since the process start. Monitor locks are commonly acquired by using the lock keyword in C#, or by calling Monitor.Enter() and Monitor.TryEnter().                                                  | `{contentions}`   | ObservableCounter | `Int64`    |                  |                  |
```

Comment:
I'll also update this to `lock_conventions.count`

```suggestion
| process.runtime.dotnet.**monitor.lock_contentions.count**     | The number of times there was contention when trying to acquire a monitor lock since the process start. Monitor locks are commonly acquired by using the lock keyword in C#, or by calling Monitor.Enter() and Monitor.TryEnter().                                                  | `{contentions}`   | ObservableCounter | `Int64`    |                  |                  |
```